ch2.txt

```diff
@@ -2,13 +2,6 @@ CHAPTER 2. WAGING WAR
 
 1.  Sun Tzu said: In the operations of war, where there are in the field a thousand swift chariots, as many heavy chariots, and a hundred thousand mail-clad soldiers, with provisions enough to carry them a thousand LI, the expenditure at home and at the front, including entertainment of guests, small items such as glue and paint, and sums spent on chariots and armor, will reach the total of a thousand ounces of silver per day. Such is the cost of raising an army of 100,000 men. 
 
-2. When you engage in actual fighting, if victory is long in coming, then men's weapons will grow dull and their ardor will be damped. If you lay siege to a town, you will exhaust your strength. 
-
-3. Again, if the campaign is protracted, the resources of the State will not be equal to the strain. 
-
-4. Now, when your weapons are dulled, your ardor damped, your strength exhausted and your treasure spent, other chieftains will spring up to take advantage of your extremity. Then no man, however wise, will be able to avert the consequences that must ensue. 
-
-5. Thus, though we have heard of stupid haste in war, cleverness has never been seen associated with long delays. 
```

Comment:
Nice edits. Improves flow. However, have you considered:

```diff
-2. When you engage in actual fighting, if victory is long in coming, then men's weapons will grow dull and their ardor will be damped. If you lay siege to a town, you will exhaust your strength. 
+2. When you engage in actual fighting, if victory is long in coming, then men's weapons will grow dull and their ardor will be damped. If you lay siege to a town, you will exhaust your strength. Thus spake Zarathustra.
```

Do not give in to astonishment, and do not let your ardor be damped.